UnitTests/GitUI.Tests/UserControls/FilterInfoTests.cs

```diff
@@ -0,0 +1,712 @@
+﻿using System;
+using System.Collections.Generic;
+using ApprovalTests;
+using FluentAssertions;
+using GitCommands;
+using GitUI.UserControls.RevisionGrid;
+using LibGit2Sharp;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace GitUITests.UserControls
+{
+    [TestFixture]
+    public class FilterInfoTests
+    {
+        [Test]
```

Comment:
As well as changes in the approval files
```suggestion
        [SetCulture("en-US")]
        [SetUICulture("en-US")]
        [Test]
```